x/expiration/client/cli/cli_test.go

```diff
@@ -0,0 +1,828 @@
+	metadatacli "github.com/provenance-io/provenance/x/metadata/client/cli"
+	metadatatypes "github.com/provenance-io/provenance/x/metadata/types"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type IntegrationCLITestSuite struct {
+	suite.Suite
+
+	cfg             testnet.Config
+	testnet         *testnet.Network
+	keyring         keyring.Keyring
+	keyringDir      string
+	keyringAccounts []keyring.Record
+
+	asJson string
+	asText string
+
+	acctErr error
+
+	accountAddr    sdk.AccAddress
+	accountAddrStr string
+
+	user1Addr    sdk.AccAddress
+	user1AddrStr string
+
+	user2Addr    sdk.AccAddress
+	user2AddrStr string
+
+	user3Addr    sdk.AccAddress
+	user3AddrStr string
+
+	user4Addr    sdk.AccAddress
+	user4AddrStr string
+
+	user5Addr    sdk.AccAddress
+	user5AddrStr string
+
+	user6Addr    sdk.AccAddress
+	user6AddrStr string
+
+	userOtherAddr sdk.AccAddress
+	userOtherStr  string
+
+	moduleAssetID1 string
+	moduleAssetID2 string
+	moduleAssetID3 string
+	moduleAssetID4 string
+	moduleAssetID5 string
+	moduleAssetID6 string
+
+	sameOwner string
+	diffOwner string
+
+	time    time.Time
+	deposit sdk.Coin
+	// signers []string
+
+	scopeID metadatatypes.MetadataAddress
+
+	noExpScopeId metadatatypes.MetadataAddress
+
+	expiration1 expirationtypes.Expiration
+	expiration2 expirationtypes.Expiration
+	expiration3 expirationtypes.Expiration
+	expiration4 expirationtypes.Expiration
+	expiration5 expirationtypes.Expiration
+	expiration6 expirationtypes.Expiration
+
+	expiration1AsJson string
+	expiration2AsJson string
+	expiration3AsJson string
+
+	expiration1AsText string
+	expiration2AsText string
+	expiration3AsText string
+}
+
+func TestIntegrationCLITestSuite(t *testing.T) {
+	suite.Run(t, new(IntegrationCLITestSuite))
+}
+
+func (s *IntegrationCLITestSuite) SetupSuite() {
+	s.T().Log("setting up integration test suite")
+	pioconfig.SetProvenanceConfig("", 0)
+	cfg := testutil.DefaultTestNetworkConfig()
+
+	cfg.NumValidators = 1
+	genesisState := cfg.GenesisState
+	s.cfg = cfg
+	s.generateAccountsWithKeyrings(7)
+
+	// An account
+	s.accountAddr, s.acctErr = s.keyringAccounts[0].GetAddress()
+	s.Require().NoError(s.acctErr, "getting keyringAccounts[0] address")
+	s.accountAddrStr = s.accountAddr.String()
+
+	// A user account
+	s.user1Addr, s.acctErr = s.keyringAccounts[1].GetAddress()
+	s.Require().NoError(s.acctErr, "getting keyringAccounts[1] address")
+	s.user1AddrStr = s.user1Addr.String()
+
+	// A second user account
+	s.user2Addr, s.acctErr = s.keyringAccounts[2].GetAddress()
+	s.Require().NoError(s.acctErr, "getting keyringAccounts[2] address")
+	s.user2AddrStr = s.user2Addr.String()
+
+	// A third user account
+	s.user3Addr, s.acctErr = s.keyringAccounts[3].GetAddress()
+	s.Require().NoError(s.acctErr, "getting keyringAccounts[3] address")
+	s.user3AddrStr = s.user3Addr.String()
+
+	// A third user account
```

Comment:
Probably just remove these comments. Or if keeping them, update the rest to the correct numbers.